setup.py

```diff
@@ -45,13 +45,16 @@
         'Framework :: Django :: 2.2',
         'Framework :: Django :: 3.0',
         'Framework :: Django :: 3.1',
+        'Framework :: Django :: 3.2',
         'Framework :: Django :: 4.0',
+        'Framework :: Django :: 4.1',
         'Programming Language :: Python',
         'Programming Language :: Python :: 3.5',
         'Programming Language :: Python :: 3.6',
```

Comment:
Python 3.5 and 3.6 have both reached EOL, and aren't tested on CI. Should we remove it from the list? We probably ought to [set the `python_requires`](https://packaging.python.org/en/latest/guides/dropping-older-python-versions/#specify-the-version-ranges-for-supported-python-distributions) as well.